osu.Game.Tournament/Screens/Editors/TeamEditorScreen.cs

```diff
@@ -277,7 +277,8 @@ private void load()
                         userId.Value = user.Id.ToString();
                         userId.BindValueChanged(idString =>
                         {
-                            int.TryParse(idString.NewValue, out var parsed);
+                            if (!(int.TryParse(idString.NewValue, out var parsed)))
```

Comment:
won't this mean that a panel stays looking like it is a valid use even though the id has changed?

Reply:
Right, replaced this with fixing the user ID on a successful request instead.